app/policies/wiki_policy.rb

```diff
@@ -1,3 +1,79 @@
 class WikiPolicy < ApplicationPolicy
+  attr_reader :user, :wiki
 
+  def initialize(user, wiki)
+    @user = user
+    @wiki = wiki
+  end
+
+  def create?
+    user.admin? || user.standard? || user.premium?
+  end
+
+  def new?
+    create?
+  end
+
+  def update?
+    user.present?
```

Comment:
For this one, I might check to see if the user has the right to update. They'd need to be present, and either the record owner or admin (until you do collaborators, then one of those too).